parley/src/resolve/mod.rs

```diff
@@ -140,7 +141,9 @@ impl ResolveContext {
             StyleProperty::FontWeight(value) => FontWeight(*value),
             StyleProperty::FontVariations(value) => FontVariations(self.resolve_variations(value)),
             StyleProperty::FontFeatures(value) => FontFeatures(self.resolve_features(value)),
-            StyleProperty::Locale(value) => Locale(value.map(Language::parse).flatten()),
+            StyleProperty::Locale(value) => {
+                Locale(value.and_then(|v| LanguageIdentifier::try_from_str(v).ok()))
```

Comment:
consider documenting which type of locale this is. `LanguageIdentifier` does not support full BCP-47 locales; you might want to parse as a `Locale` and then discard everything but the `id`.

this is preexisting, however, the diff here is unnecessary I think because `fontique::Language` is `LanguageIdentifier`.